sonar-scala-plugin/src/test/java/org/sonarsource/scala/converter/FunctionDeclarationTreeTest.java

```diff
@@ -64,4 +65,43 @@ public void function_without_parameter_list() {
   public void function_with_multiple_parameter_lists() {
```

Comment:
Could you add assertion on formal parameters when using multiple parameter lists?